src/extension/VSProject.cs

```diff
@@ -241,16 +241,41 @@ private void Load()
         /// </summary>
         /// <returns>True if the project was successfully loaded, false otherwise.</returns>
         private bool TryLoadDotNetCoreProject()
-        {
+        {
             XmlNode root = _doc.SelectSingleNode("Project");
 
             if (root != null && SafeAttributeValue(root, "Sdk") != null)
             {
                 string targetFramework = _doc.SelectSingleNode("Project/PropertyGroup/TargetFramework").InnerText;
 
                 XmlNode assemblyNameNode = _doc.SelectSingleNode("Project/PropertyGroup/AssemblyName");
-                // Even console apps are dll's even if <OutputType> has value 'EXE'
-                string assemblyName = assemblyNameNode == null ? $"{Name}.dll" : $"{assemblyNameNode.InnerText}.dll";
+                
+                // Even console apps are dll's even if <OutputType> has value 'EXE',
+                // if TargetFramework is netcore
+                string outputType = "dll";
+
+                if (!targetFramework.StartsWith("netcore"))

```

Comment:
This may be a non-issue, but what about netstandard projects? Does this fix need to be specific to net20-net472 target frameworks, rather than "anything except netcore"? (I'm not sure on the answer here - but don't want this change to affect other functionality!)

Reply:
Yes. I wasn't sure on what actual test to do here.
Really I'd want to test for 'net[0-9]' but I think I was being lazy.
I'll make improve it in some way